### ISSUE TITLE
Feature Request: flag for disabling automatic add of default Helm repos

### ISSUE THREAD

**Issue by jmontroy90** [Praqma/helmsman#301]
We work in an extremely constrained networking environment both locally and in dev/qa/prod. As such, we are permanently unable to resolve these default Helm repos (from `main.go`):
```
const stableHelmRepo = "https://kubernetes-charts.storage.googleapis.com"
const incubatorHelmRepo = "http://storage.googleapis.com/kubernetes-charts-incubator"
```
I would like to add a flag to disable the automatic add of these default Helm repos. I'm more than happy to implement what I anticipate to be a minor change, but wanted to open it up for discussion first.

The PR would add a conditional to the top of `utils.go:addDefaultHelmRepos`, returning early if the flag was set.

Alternatives are currently to manually override both "stable" and "incubator" in every single desired state file. This seems excessive, and not possible in cases where internal artifact repos are not available (we don't have an "incubator" equivalent, for example). Adding this flag and conditional in one spot will future-proof against name changes and newly-added repos.

This is related, but slightly different: https://github.com/Praqma/helmsman/issues/294

Thanks!

**Comment by luisdavim** [comment]
Adding a flag to toggle off the default repos seems reasonable.

**Comment by jmontroy90** [comment]
@sami-alajrami are you able to assist on potentially cutting a release?

**Comment by luisdavim** [comment]
this is available in tag `v1.12.1`
